addon/PreviewProvider.js

```diff
@@ -370,7 +378,7 @@ PreviewProvider.prototype = {
    */
   insertMetadata(links, metadataSource) {
     const linksToInsert = links.map(link => Object.assign({}, link, {
-      expired_at: (this.options.metadataTTL) + Date.now(),
+      expired_at: (this._getMetadataTTL()) + Date.now(),
```

Comment:
@ncloudioj I agree but for the purposes of the experiment this is most consistent than statefully mirroring the experiment state and setting a local constant.  When our experimentation is complete we will move it back to being a hard coded constant.